### PR TITLE
Sort out the use of flexbox on the large navigation menu.

### PR DESCRIPTION
**Pull request by JayyajGH**
After investigating the IE11 issue I found a few oddities with the way the navbar was laid out.  The centre toggle with a 100% width and centre justification and the right hand nav with an auto left margin just seemed to be confusing the hell out of flexbox. I think it was probably the addition of the centre toggle that broke this in IE11.

I've just removed all the weird stuff and told the nav parent to justify the content.    I've not touched the mobile navigation.

I've tested this in IE11 on browserstack and in devtools in Chrome so this will still need a proper test on different browsers/devices.  If I don't get a chance to do this today then I'll do it tomorrow.

I thought I'd check this in as you could test it if you were desperate for a fix!

### CODE REVIEW

**Comment by JayyajGH** [comment]
Right I've now tested this on a few browsers and it seems ok.  Are you easily able to test it on IE11?  I've only tested it on IE11 in devtools and it looks ok.

**Comment by edwh** [comment]
Looks ok on Chrome, Edge, FF.  On IE11 the logo is positioned wrongly and the hamburger disappears:

image

The centering of "Communities" looks a bit off though that's an existing problem.

**Comment by JayyajGH** [comment]
Hmmm that's odd as it seemed to work ok in detools.  What do I need to do to get a build to work on IE11 in browserstack?

**Comment by edwh** [comment]
I'm using IE11 direct on my W10 home machine. What OS are you on?

**Comment by JayyajGH** [comment]
Actually I've recreated it.    I only fixed it when it was on larger screens.  It looks like my fix has worked for what I intended but I also need to do something about the mobile view too.

I'm on Windows 10.  Oh damn. IE11 is available on W10 isn't it.  I forgot and I thought it was just Edge...

**Comment by JayyajGH** [comment]
I'm just heading out so I'll take a look tomorrow. If you're happy with it on larger screens then this change is pretty standalone if you wanted to put it out which probably fixes the majority of IE11 use.  A very brief look at the mobile version might be a bit more refactoring work.

**Comment by JayyajGH** [comment]
Just testing a fix for this now but about to head into a meeting...

**Comment by JayyajGH** [comment]
Right.  See what you think of that...  I've basically just removed the auto margins and converted it to use flexbox.

File: layouts/default.vue [hunk 1]
```diff
@@ -139,69 +137,69 @@
...
+              </div>
+            </b-nav-item>
+          </client-only>
+        </b-navbar-nav>
 
+        <b-navbar-nav class="">
+          <b-btn v-if="loggedIn" v-b-toggle.nav_collapse_mobile class="toggler white">
+            <v-icon name="bars" class="mb-1" scale="1.5" />
+          </b-btn>
+        <!--        <b-navbar-toggle v-if="loggedIn" target="nav_collapse_mobile" class="navbar-dark" />-->
```
Comment (aaronferris) [inline layouts/default.vue]: Any point in this code if we're commenting it out? Looks like legacy but id be tempted to remove it, its just noise

Comment (JayyajGH) [reply layouts/default.vue]: @edwh - Can this be removed or is it a reminder for something?

Comment (edwh) [reply layouts/default.vue]: It can be removed.

Comment (JayyajGH) [reply layouts/default.vue]: Done. :-)

**Comment by edwh** [comment]
Looks good on Chrome, FF, Edge, IE11.  Will merge.  Thanks.

**Pull request merged**